src/rgw/rgw_rados.cc

```diff
@@ -6306,77 +6306,460 @@ int RGWRados::Object::Read::read(int64_t ofs, int64_t end, bufferlist& bl, optio
   return bl.length();
 }
 
-struct get_obj_data {
-  RGWRados* store;
-  RGWGetDataCB* client_cb;
-  rgw::Aio* aio;
```

Comment:
it looks like this reverts the `AioThrottle` work in https://github.com/ceph/ceph/pull/25208, https://github.com/ceph/ceph/pull/26173, and https://github.com/ceph/ceph/pull/26461

Reply:
refactored